lib/index.js

```diff
@@ -32,15 +34,30 @@ var log = logMaker('lib/index.js');
 
   var userApiWatch = hakkenClient.watchFromConfig(config.userApi.serviceSpec);
   userApiWatch.start();
-  // metrics
+
   var metricsWatch = hakkenClient.watchFromConfig(config.metrics.serviceSpec);
   metricsWatch.start();
 
+  //Parts required to initialise the gatekeeper client
+  var gatekeeperApiWatch = hakkenClient.watchFromConfig(config.gatekeeper.serviceSpec);
+  gatekeeperApiWatch.start();
+  var httpClient = amoeba.httpClient();
+  var userApiClient = require('user-api-client').client(config.userApi, userApiWatch);
+
   var service = require('./seagullService.js')(
     config,
     require('./mongoCrudHandler.js')(mongoConfig),
-    require('user-api-client').client(config.userApi, userApiWatch),
-    require('user-api-client').metrics(metricsWatch, config, log)
+    userApiClient,
+    require('user-api-client').metrics(
+      metricsWatch,
+      config,
+      log
+    ),
+    require('tidepool-gatekeeper').client(
```

Comment:
so funnily I did that and then changed it back to keep the things together that came from the user-api :) Will happily change